sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample5_RecognizePiiEntitiesAsync.cs

```diff
@@ -15,7 +15,7 @@ public async Task RecognizePiiEntitiesAsync()
             string endpoint = TestEnvironment.Endpoint;
             string apiKey = TestEnvironment.ApiKey;
 
-            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey), CreateSampleOptions());
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey), CreateSampleOptions(TextAnalyticsClientOptions.ServiceVersion.V2022_03_01_Preview));
```

Comment:
Strange not to see any kind of snippet anywhere in this one.   Should we consider removing it?

Reply:
I'm going to leave it as-is for now. Let me know what you think and if we would like to make this change,I will create a wide issue as it will impact all of our samples.